components/BaseHeadline.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <h2
+    v-if="title"
```

Comment:
Is the v-if useful here? Your prop is required anyways. So I wouild not expect it to not be existing